setup.py

```diff
@@ -236,15 +236,17 @@ def cython_extensions_and_necessity():
     'protobuf>=3.3.0',
 )
 
-if not PY3:
-  INSTALL_REQUIRES += ('futures>=2.2.0', 'enum34>=1.0.4')
-
 SETUP_REQUIRES = INSTALL_REQUIRES + (
     'sphinx>=1.3',
     'sphinx_rtd_theme>=0.1.8',
     'six>=1.10',
   ) if ENABLE_DOCUMENTATION_BUILD else ()
 
+EXTRAS_REQUIRE = {
+  ':python_version<"3.0"': ('futures>=2.2.0',),
```

Comment:
Not that it matters in practice, but the `concurrent.futures` package was introduced in Python 3.2, not 3.0.

Reply:
OTOH, `futures` only works on Python 2 so disregard me :)